darts/models/forecasting/regression_model.py

```diff
@@ -646,6 +646,49 @@ def _prepare_quantiles(quantiles):
 
         return quantiles, median_idx
 
+    def _predict_quantiles(
+        self, superfun: Callable, num_samples: int, **kwargs
+    ) -> Union[TimeSeries, List[TimeSeries]]:
+        predictions = []
+        for quantile, fitted in self._model_container.items():
+            self.model = fitted
+            prediction = superfun(**kwargs)
+            if not isinstance(prediction, list):  # handles the single series case
```

Comment:
This would be a bit more general
```suggestion
            if not isinstance(prediction, Sequence):  # handles the single series case
```